### PR TITLE
DOC-12379: 3.1.9 relnotes

### PR DESCRIPTION
**Pull request by ElliotFrancisHunter**
Release notes for Sync Gateway 3.1.9 release.

### CODE REVIEW

**Review by RichardSmedley — APPROVED** [review]
:+1:

File: modules/ROOT/pages/_partials/release_notes/sync-gateway-3-1-9-release-note.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
+== 3.1.9 -- July 2024
+
+Version 3.1.9 of Sync Gateway delivers the following features and enhancements:
```
Comment (bfavini) [inline modules/ROOT/pages/_partials/release_notes/sync-gateway-3-1-9-release-note.adoc:3]: All of these tickets are private. Please verify whether they should remain private--and delete any private tickets from the release ntoes.

**Review by bfavini — COMMENTED** [review]
Please verify that no tickets referenced in the RNs are private in JIRA. If they must remain private, remove them from the release notes.

**Comment by adamcfraser** [comment]
Looks good to me - I've updated the referenced tickets to be public.  We normally keep tickets private until ready to release - thanks Elliot for the reminder to switch these over.

**Comment by adamcfraser** [comment]
I've just noticed one ticket that shouldn't be included in the release notes - adding a comment now.

File: modules/ROOT/pages/_partials/release_notes/sync-gateway-3-1-9-release-note.adoc [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+[#maint-3-1-9]
+=== Fixed Issues
+
+* https://issues.couchbase.com/browse/CBG-3934[CBG-3934 -  Fixed rotated_logs_size_limit not used by ConsoleLogger's FileOutput]
```
Comment (adamcfraser) [inline modules/ROOT/pages/_partials/release_notes/sync-gateway-3-1-9-release-note.adoc]: This ticket has been moved to 3.1.10, should be removed from release notes.

**Review by adamcfraser — DISMISSED** [review]

File: modules/ROOT/pages/_partials/release_notes/sync-gateway-3-1-9-release-note.adoc (hunk 2 above)
Comment (ElliotFrancisHunter) [reply modules/ROOT/pages/_partials/release_notes/sync-gateway-3-1-9-release-note.adoc]: Removed

**Pull request merged**